en/core-api/advanced-features.md

```diff
@@ -1212,34 +1527,48 @@ bill_info8 | String<br>(***optional***) | Value for Label 4. Mandiri only allows
+Virtual Account number displayed to customer contains two parts; the company code and the unique code. The table given below, explains this with an example.
 
-* Only digits are allowed.
-* Different banks have different specs on their custom VA numbers. Please see the documentation on the respective banks.
-* If the number provided is already utilized for another order, then a different unique number will be used instead.
-* If the number provided is longer than required, then the unnecessary digits in the end will be trimmed.
-* If the number provided is shorter than required, then the number will be prefixed with zeros.
+| Virtual Account Number | Company Code | Unique Code  |
+| ---------------------- | ------------ | ------------ |
+| `{91012}{12435678}`    | `{91012}`    | `{12435678}` |
 
-Note: On Production mode, not all Bank support custom VA number, it depends on the agreement, please consult with Midtrans Activation team for further info.
+The unique code can be customized.
 
-## Consideration and Limitation
-By using Midtrans API there are some consideration and limitation you need to keep in mind, that will be explained below.
+Rules for customizing Virtual Account number is given below.
+
+* Only digits are allowed.
+* Different banks have different specifications on their custom VA numbers. Please see the documentation of the respective banks.
+* If the number provided is already utilized for another order, then a unique number will be used instead.
+* If the number provided is longer than required, then the unnecessary digits in the end will be truncated.
+* If the number provided is shorter than required, then the number will be prefixed with zeros.
 
-### Maximum Request Size Limit
+?>***Note***: In *Production* environment, all banks do not support custom VA number. It depends on the agreement. Please consult with Midtrans Activation team for further information.
 
-Midtrans API allow maximum size of **16kb** per request (**\~16000 total characters**). Please strive to keep it under this limit to avoid request failure.
+## Consideration and Limitations
+There are a few limitations to consider while using Midtrans API. These limitations are given below.
 
-Tips: You can try to limit the number of `item_details` from your request, or atleast group it into fewer (or 1 generic) `item_details`.
+### 1. Maximum Request Size Limit
+Midtrans API allows a maximum size of 16kb or 16000 total characters per request. Please try to keep it under the limit to avoid request failures.
 
-### Card Token ID Expiry Time
+?>***Tips***: Limit the number of `item_details` from your request, or group it into fewer `item_details`.
 
-For regular card transaction, card's `token_id` (for non-recurring, non-one-click, non-two-click token) and also the 3ds `redirect_url` lifetime is **10 minutes**. 
+### 2. Card Token ID Expiry Time
```

Comment:
I don't think this should be numbered. As this is just a list of limitation with no significant order, and is not sequential.